build.sh

```diff
@@ -60,9 +60,13 @@ function ensureCMakeRan {
     mkdir -p "${LIBRMM_BUILD_DIR}"
     if (( RAN_CMAKE == 0 )); then
         echo "Executing cmake for librmm..."
-        cmake -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
+        cmake -GNinja \
```

Comment:
I thought we were going to change this so that it is configurable (some users might not have Ninja installed). @robertmaynard ?

Reply:
Good point. Which do you prefer to be the default build system?

Reply:
I use Ninja, but I don't think we can count on everyone to agree with that. I think that Ninja is less common than make.

Reply:
We should drop the explicit generator, and instead allow selection to happen via the `CMAKE_GENERATOR` environment variable ( https://cmake.org/cmake/help/latest/envvar/CMAKE_GENERATOR.html ).